karma.conf.js

```diff
@@ -0,0 +1,79 @@
+// Karma configuration
+module.exports = function(config) {
+    config.set({
+        // base path that will be used to resolve all patterns (eg. files, exclude)
+        basePath: 'normandy/control/tests/',
+
+        // frameworks to use
+        // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+        frameworks: ['mocha'],
```

Comment:
I'm fine with mocha, but want to note that normandy-actions is using Jasmine currently (because I had issues with sinon and webpack). It'd be nice to be consistent, but I'm also fine with sticking with mocha if we like it better (and possibly switching normandy-actions back to mocha if we can figure out the issues with sinon).